main/tests.py

```diff
@@ -87,3 +88,21 @@ def test_unlogged_user_access_to_homepage(self, client):
         response = client.get("/homepage/")
         assert response.status_code == 302
         assert response['Location'] == '/login/?next=/homepage/'
+
+
+@pytest.mark.django_db
+class TestOrdersView:
+    def test_unlogged_user_access_to_orders(self, client):
```

Comment:
I think we can trust Django's `login_required` decorator functionality

Reply:
Removed.